metron-platform/metron-elasticsearch/pom.xml

```diff
@@ -184,7 +184,8 @@
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <version>${global_httpclient_version}</version>
+            <!--version>${global_httpclient_version}</version-->
+            <version>4.5.2</version>
```

Comment:
Is this needed?

Reply:
This was a mistake.  I had to change it to 4.5.2 to get the tests to run in my IDE but the tests work fine otherwise with our global version.  I don't think our global version is correct here but will leave it for now as it's out of scope for this PR.